roles/fio/files/runfio.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+declare -A args
+
+#read all args in key value pairs
+for arg in "$@"; do
+    # Check if the argument contains an equal sign
+    if [[ "$arg" == *"="* ]]; then
+        # Split the argument into key and value
+        key=${arg%%=*}
+        value=${arg#*=}
+        # Store the key-value pair in the associative array
+        args["$key"]="$value"
+    else
+        echo "Invalid argument: $arg (should be in key=value format)"
+    fi
+done
```

Comment:
Could we switch to `getopts`? For example
```
LONGOPTS=device:,time:,num:,verbose
OPTIONS=d:t:n:v

_args=$(getopt --options=${OPTIONS} --longoptions=${LONGOPTS} --name "$@" -- "$@")
while true; do
  case "$1" in
    -d|--device)
      device=$2
      shift 2
      ;;
    -t|--time)
      time=$2
      shift 2
      ;;
    --)
      shift
      break
      ;;
    *)
      echo "Unknown option"
      exit 3
      ;;
  esac
done
```

Reply:
Now uses getopt